app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/pixels/DeviceShieldPixelNames.kt

```diff
@@ -225,5 +225,8 @@ enum class DeviceShieldPixelNames(override val pixelName: String, val enqueue: B
 
     ATP_REPORT_ALWAYS_ON_ENABLED_DAILY("m_atp_ev_always_on_enabled_d"),
     ATP_REPORT_ALWAYS_ON_LOCKDOWN_ENABLED_DAILY("m_atp_ev_always_on_lockdown_enabled_d"),
+
+    ATP_REPORT_UNPROTECTED_APPS_BUCKET("m_atp_unprotected_apps_bucket_%d"),
```

Comment:
This was the agreed name. I noticed there is a convention for the pixels. Should I rename it to respect the convention? This is a temporary pixel.